config/filter.d/mongodb-auth.conf

```diff
@@ -0,0 +1,28 @@
+# Fail2Ban filter for unsuccesfull MongoDB authentication attempts
+#
+#
+
+[Init]
+maxlines = 10
+
+[Definition]
+failregex = ^\s+\[conn(?P<__connid>\d+)\] Failed to authenticate [^\n]*<SKIPLINES>\s+\[conn(?P=__connid)\] end connection <HOST>
+
+ignoreregex =
+
+
+# DEV Notes:
+#
+# Log example:
+# 2016-10-20T09:54:27.108+0200 [initandlisten] connection accepted from 127.0.0.1:53276 #1 (1 connection now open)
```

Comment:
Imho, the dev notes should describe what the regex does, not the examples of log, because they could be found in the test cases